webpack.common.config.ts

```diff
@@ -135,6 +135,9 @@ module.exports = {
     ),
     topic_viewer:
       commonPrefix + '/pages/topic-viewer-page/topic-viewer-page.import.ts',
+    voiceover_admin:
+      commonPrefix + '/pages/voiceover-admin-page/' +
+      'voiceover-admin-page.import.ts',
```

Comment:
@Nik-09 Can you please avoid adding items to webpack as we are working on removing these files, you can follow the steps mentioned in #19435 to remove it!

Reply:
Thank you @DubeySandeep for pointing this out, I was unaware of this change. In the upcoming PR (ETA 1st March) of the voiceover admin page, I will migrate this page to the Angular module.

Reply:
I'm tracking this over here: https://github.com/oppia/oppia/issues/19435#:~:text=webpack%20section)-,Voiceover%20admin,-%40Nik%2D09%20%5BExpected

Please leave any updates on the issue thread (like for any date changes etc.)!

Reply:
@Nik-09 Any update? Please update the issue #19435 once you create the PR!

Reply:
Hi, @DubeySandeep I have updated the issue description to 5th March. Thank you